docs/docs/development/hardware-integration/physical-layouts.md

```diff
@@ -0,0 +1,429 @@
+
+![A 4x5 numpad/macropad](../../assets/hardware-integration/numpad.svg)
+
+Let us first consider each side individually. The "reference" position map of the left side would look like this:
+
+```dts
+macropad_map: macropad {
+    physical-layout = <&macropad_layout>;
+    positions
+        = < 0  1  2  3>
+        , < 4  5  6  7>
+        , < 8  9 10 11>
+        , <12 13 14 15>
+        , <16 17 18 19>;
+};
+```
+
+Meanwhile, the "reference" position map of the right side with fewer keys would look like this:
+
+```dts
+numpad_map: numpad {
+    physical-layout = <&numpad_layout>;
+    positions
+        = < 0  1  2  3>
+        , < 4  5  6  7>
+        , < 8  9 10   >
+        , <11 12 13 14>
+        , <15    16   >;
+};
+```
+
+As a reminder, the `positions` property is a one-dimensional array like the `keys` property, formatted nicely through the use of whitespace and angle bracket groupings.
+
+If the left side with more keys was used as the reference layout, then the overall position map of the keyboard would look like this:
+
+```dts
+/ {
+    keypad_lossless_position_map {
+        compatible = "zmk,physical-layout-position-map";
+        complete;
+
+        macropad_map: macropad {
+            physical-layout = <&macropad_layout>;
+            positions
+                = < 0  1  2  3>
+                , < 4  5  6  7>
+                , < 8  9 10 11>
+                , <12 13 14 15>
+                , <16 17 18 19>;
+        };
+
+        numpad_map: numpad {
+            physical-layout = <&numpad_layout>;
+            positions
+                = < 0  1  2  3>
+                , < 4  5  6  7>
+                , < 8  9 10 17>
+                , <11 12 13 14>
+                , <15 18 16 19>;
+        };
+    };
+};
+```
+
+The "missing" positions are filled with the "spare" numbers of the layout with more keys. Meanwhile, if the right side with fewer keys is used as a reference, then the overall position map would look like this:
+
+```dts
+/ {
+    keypad_lossy_position_map {
+        compatible = "zmk,physical-layout-position-map";
+        complete;
+
+        macropad_map: macropad {
+            physical-layout = <&macropad_layout>;
+            positions
+                = < 0  1  2  3>
+                , < 4  5  6  7>
+                , < 8  9 10   >
+                , <12 13 14 15>
+                , <16    18   >;
+        };
+
+        numpad_map: numpad {
+            physical-layout = <&numpad_layout>;
+            positions
+                = < 0  1  2  3>
+                , < 4  5  6  7>
+                , < 8  9 10   >
+                , <11 12 13 14>
+                , <15    16   >;
+        };
+    };
+};
+```
+
+The above example is "lossy" because (unlike the "lossless" example) if a user switches from the macropad layout to the numpad layout _and then_ switches from the numpad layout back to the macropad layout, the assignments to the keys present but not listed in the macropad's map are lost.
```

Comment:
```suggestion
The above example is "lossy" because (unlike the previous "lossless" example) if a user switches from the macropad layout to the numpad layout _and then_ switches from the numpad layout back to the macropad layout, the assignments to the keys present but not listed in the macropad's map are lost.
```
The only place lossless was mentioned was in the node name, so this probably helps